project/Settings.scala

```diff
@@ -50,7 +50,7 @@ object Settings {
     val unboundId    = "3.1.1"
     val jwt          = "0.8.1"
     val slf4j        = "1.7.21"
-    val knobs        = "3.8.1a"
+    val knobs        = "3.12.27a"
 
```

Comment:
Did you try `http4s-0.15.0`? It was released yesterday but I was wondering whether the build would break with the that version.